packages/sdk/src/types/nft/mint/preprocess-meta.ts

```diff
@@ -0,0 +1,42 @@
+import type { MetaContentRepresentation } from "@rarible/api-client/build/models/MetaContent"
+
+export type PreprocessMeta = {
+	name: string
+	decimals: number
```

Comment:
Let's skip for now what's not used in rarible meta data
There's only name, description, attributes, image and animation
We don't need to preprocess any other information

Also, there should be the same preprocess function for ethereum and flow